src/hackloop.js

```diff
@@ -2,11 +2,34 @@
  * loop against the target.
  */
 
+import { displayHelp } from './libs/common.js';
+
+const FLAGS_DEF = [
+  ['bypassHack', false, 'True to exit the script instead of hack'],
+  ['help', false, 'Displays this help message'],
+];
```

Comment:
I used help text for my test case of a common method that could be consumed across multiple scripts.